bootstrap.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set +x
+
+# Ask the user for the installation directory
+read -p "Enter the installation directory (e.g., '.' or 'docs'): " install_directory
+
+# Clone the starter pack repository to a temporary directory
+echo "Cloning the starter pack repository..."
+git clone --depth 1 https://github.com/canonical/sphinx-docs-starter-pack temp-starter-pack
+rm -rf temp-starter-pack/.git
+
+# Update file contents for install directory
+echo "Updating working directory in workflow files..."
+sed -i "s|working-directory:\s*'\.'|working-directory: '$install_directory'|g" "temp-starter-pack/.github/workflows"/*
+echo "Updating .readthedocs.yaml configuration..."
+sed -i "s|configuration:\s*conf\.py|configuration: $install_directory/conf.py|g" "temp-starter-pack/.readthedocs.yaml"
+sed -i "s|requirements:\s*\.sphinx/requirements\.txt|requirements: $install_directory/.sphinx/requirements.txt|g" "temp-starter-pack/.readthedocs.yaml"
+
+# Create the specified installation directory
+echo "Creating the installation directory: $install_directory"
+mkdir -p "$install_directory"
+
+# Copy the contents of the starter pack repository to the installation directory
+echo "Copying contents to the installation directory..."
+cp -R temp-starter-pack/* temp-starter-pack/.??* "$install_directory"
+
+# Move workflow files and configuration
+if [ "$install_directory" != "." ]; then
+    echo "Moving workflow files and configuration..."
+    mkdir -p .github/workflows
+    mv "$install_directory/.github/workflows"/* .github/workflows
+    mv "$install_directory/.github/.jira_sync_config.yaml" .github/
+fi
+
+# Clean up
+echo "Cleaning up..."
+rm -rf temp-starter-pack
```

Comment:
I don't think this is an issue since we also create this folder as part of the script - I think it's worse to leave a temporary folder behind.
In Python, there's a way to create a temporary folder ensuring that it is unique. Is there something like this in bash as well? So we can be 100% sure we don't delete anything that existed before.

Reply:
I've added a timestamp which guarantees uniqueness beyond reasonable doubt.